app/features-json/view_models/view_model.rb

```diff
@@ -67,7 +67,7 @@ def viewmodel_from!(object:)
         raise "Override base_model(model) in order to use the ViewModel mixin." if @base_model.nil?
```

Comment:
I think we should remove the named parameter `object:` and just keep it with `object`, that will clean things up and does not add any useful information. What do you think?

Reply:
Not sure i'm following what you mean, can you propose something in a follow up PR?

Reply:
@minuscorp it looks like that's already done 🎉 is there anything else that's missing from this PR?

Reply:
I think this: https://github.com/fastlane/ci/pull/566 supersedes this, but @nakhbari should decide 😄